.travis.yml

```diff
@@ -16,14 +19,10 @@ before_install:
   - conda config --add channels defaults
   - conda config --add channels bioconda
   - conda config --add channels conda-forge
-  # Updating conda to the latest version may lead to speed improvements in the resolver.
-  - conda update -n base -c defaults conda
 
 install:
-  - conda env create -n my_env -f environment.yml
-  - source activate my_env
   - conda install cromwell
-  - python3 -m pip install pytest-workflow pysam
```

Comment:
Isn't pysam necessary here?

Reply:
whoops. This is an artifact from removing the conda stuff. (Copy paste from other repo)